### PR TITLE
LPS-152920 extract common method to a util on portal test to let several tests use them and reuse it

### PR DESCRIPTION
**Pull request by AliciaGarciaGarcia**
- LPS-152920 extract common method to a util on portal test to let several tests use them and reuse it
- LPS-152920 use new method instead private or protected ones

### CODE REVIEW

**Comment by liferay-lima** [comment]
ci:test:sf

**Comment by liferay-lima** [comment]
ci:test:relevant

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

**Comment by AliciaGarciaGarcia** [comment]
ci:test:sf

**Comment by AliciaGarciaGarcia** [comment]
baseline and rebase

**Comment by AliciaGarciaGarcia** [comment]
ci:test:sf

File: portal-test/src/com/liferay/portal/test/property/PortalPropertyTestUtil.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+import com.liferay.petra.reflect.ReflectionUtil;
+import com.liferay.portal.util.PropsValues;
+
+import java.lang.reflect.Field;
+import java.lang.reflect.Modifier;
+
+/**
+ * @author Alicia García
+ */
+public class PortalPropertyTestUtil {
```
Comment (adolfopa) [inline portal-test/src/com/liferay/portal/test/property/PortalPropertyTestUtil.java:26]: Why not call it `PropsValuesTestUtil`? And maybe rename the method to `setProperty`?

**Review by adolfopa — CHANGES_REQUESTED** [review]

File: portal-test/src/com/liferay/portal/test/property/PortalPropertyTestUtil.java (hunk 1 above)
Comment (AliciaGarciaGarcia) [reply portal-test/src/com/liferay/portal/test/property/PortalPropertyTestUtil.java:26]: There is another `PropsValuesTestUtil` on` src/com/liferay/portal/kernel/test/util/PropsValuesTestUtil.java`

I do not think a good idea call it the same way.

Comment (adolfopa) [reply portal-test/src/com/liferay/portal/test/property/PortalPropertyTestUtil.java:26]: Mmm, cannot we use the logic that's already there? It seems to be doing (almost) the same this.

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

**Comment by AliciaGarciaGarcia** [comment]
ci:test:sf

**Comment by Hanlf** [comment]
ci:test:relevant

**Review by adolfopa — APPROVED** [review]

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

**Comment by AliciaGarciaGarcia** [comment]
ci:test:sf

**Comment by adolfopa** [comment]
ci:forward

**Comment by AliciaGarciaGarcia** [comment]
ci:forward

**Pull request closed without merging**